index.html

```diff
@@ -1,348 +1,328 @@
-                                <li>CSS</li>
-                            </ul>
-                        </div>
-                        <div class="green-btn">
-                            <a href="single-project.html">See Project</a>
+                    <div class="item-description">
+                        <div class="item-title">
+                            <h2>Mult-Post Stories</h2>
+                            <h3>Gain + Glory</h3>
                         </div>
+                        <ul class="technologies">
+                            <li>Javascript</li>
+                            <li>Ruby on rails</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
                     </div>
-
-                    <div class="grid-item">
-                        <div class="item-image">
-                            <!-- <img src="" alt="place image here" /> -->
-                        </div>
-                        <div class="item-description">
-                            <div class="item-title">
-                                <h2>Mult-Post Stories</h2>
-                                <h3>Gain + Glory</h3>
-                            </div>
-                            <ul class="technologies">
-                                <li>Javascript</li>
-                                <li>Ruby on rails</li>
-                                <li>HTML</li>
-                                <li>CSS</li>
-                            </ul>
-                        </div>
-                        <div class="green-btn">
-                            <a href="single-project.html">See Project</a>
-                        </div>
+                    <div class="green-btn">
+                        <a href="single-project.html">See Project</a>
                     </div>
+                </div>
 
-                    <div class="grid-item">
-                        <div class="item-image">
-                            <!-- <img src="" alt="place image here" /> -->
-                        </div>
-                        <div class="item-description">
-                            <div class="item-title">
-                                <h2>Mult-Post Stories</h2>
-                                <h3>Gain + Glory</h3>
-                            </div>
-                            <ul class="technologies">
-                                <li>Javascript</li>
-                                <li>Ruby on rails</li>
-                                <li>HTML</li>
-                                <li>CSS</li>
-                            </ul>
-                        </div>
-                        <div class="green-btn">
-                            <a href="single-project.html">See Project</a>
-                        </div>
+                <div class="grid-item">
+                    <div class="item-image">
+                        <!-- <img src="" alt="place image here" /> -->
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.